src/tasks/deploy.mk

```diff
@@ -55,7 +55,7 @@ deploy-canary: ## deploy-canary: Deploy canary app to staging
 	nht configure $(VAULT_NAME) $(HEROKU_APP_STAGING)
 
 	@echo "Deploying app to $(HEROKU_APP_STAGING)"
-	@git push https://git.heroku.com/$(HEROKU_APP_STAGING).git master
+	@git push https://git.heroku.com/$(HEROKU_APP_STAGING).git HEAD
```

Comment:
**issue:** This just fixes the canary deployment right? I think you'll need to make the same change to [deploy-staging](https://github.com/Financial-Times/n-gage/blob/master/src/tasks/deploy.mk#L89)